apps/condo/domains/common/constants/currencies.js

```diff
@@ -180,6 +180,19 @@ const ISO_CODES = [
     'ZWL',
 ]
 
+const CURRENCY_SYMBOLS = {
+    CNY: '¥',
+    EUR: '€',
+    INR: 'र',
+    JPY: '¥',
+    RUB: '₽',
+    USD: '$',
+}
```

Comment:
As I remember, we were using some 3rd party library which handled mapping currency to currency symbol

Why do we need to explicitly specify CURRENCY_SYMBOLS?

Reply:
So what's the solution?